src/images/projectIcons/seamonkey.svg

```diff
@@ -1 +1 @@
1.62 4.26-17.83 4.23-3.75.14-7.43-.73-11.1-1.33-.22-2.41-.34-4.83-.37-7.25z"/><path fill="#4bacb8" d="M104.98 64.25c6.92-.91 13.95-.79 20.9-.22-.06 2.74-.28 5.48-.59 8.2-7.22-.92-13.76-4.33-20.88-5.65.14-.58.42-1.75.57-2.33z"/><path fill="#4bacb8" d="M71.34 95.91c14.31-4.67 28.67-14.13 33.07-29.33 7.12 1.32 13.66 4.73 20.88 5.65-.92 7.88-3.47 15.6-7.6 22.4-4.84-.29-8.42 3.49-12.41 5.61-8.51 4.73-16.86 9.75-25.29 14.63-6.72 3.6-13.11 7.94-20.43 10.28-9.08-.5-17.96-3.44-25.74-8.1 11.01-4.38 20.36-11.9 30.98-17.03 2.38-1.05 5.4-1.4 6.54-4.11z"/><path fill="#4bacb8" d="M32.87 68.05c6.79 2.51 13.78 4.37 20.91 5.6-8.46 13.15-19.08 25.03-31.8 34.2-10.08-9.69-16.87-22.9-18.04-36.9 3.67.6 7.35 1.47 11.1 1.33 6.21.03 12.02-2.36 17.83-4.23z"/><path fill="#4bacb8" d="M79.12 68.99c2.84 1.34 3.83 4.56 5.1 7.18-.16.14-.47.43-.63.58l-.87.75-.67.56c-1.13-.72-2.24-1.45-3.36-2.18-.19-2.31-.08-4.63.43-6.89z"/><path fill="#4bacb8" d="M88.81 70.65c.65-.75 1.3-1.49 1.97-2.22 2.23 2.18 4.31 4.49 6.32 6.86-1.69 2.8-4.18 4.98-6.57 7.16-.13-3.99-.85-7.92-1.72-11.8z"/><path fill="#4bacb8" d="M88.16 71.42l.65-.77c.87 3.88 1.59 7.81 1.72 11.8-.68.59-1.37 1.17-2.06 1.73-1.38-2.63-2.98-5.15-4.88-7.43.16-.15.47-.44.63-.58 1.41-1.5 2.64-3.15 3.94-4.75z"/><path fill="#4bacb8" d="M75.14 73.58c1.26 2.47 2.18 5.09 2.86 7.77 1.4-1.04 2.72-2.17 4.05-3.29l.67-.56c-.2 3.36-.01 6.74.45 10.07-.91.5-1.82 1-2.74 1.47-.85-2.55-1.68-5.17-3.28-7.37-.97 2.99-1.09 6.16-.61 9.24-.96.45-1.95.87-2.93 1.28-.29-2.37-.7-4.73-1.71-6.91-2.01 2.38-2.63 5.47-2.81 8.5l-1.55.48c-.21-2.06-.48-4.1-.84-6.13-3.36 1.51-6.84 2.75-10.49 3.37.68-1.35.78-3.24 2.17-4.12 4.5-2.86 9.25-5.5 12.81-9.56 1.13 2.09 1.42 4.41 1.19 6.76 1.2-.87 2.4-1.75 3.59-2.64-1.25-.61-2.48-1.22-3.72-1.84.35-2.41 1.83-4.39 2.89-6.52z"/><path fill="#4bacb8" d="M83.59 76.75c1.9 2.28 3.5 4.8 4.88 7.43-1.64 1.32-3.47 2.37-5.3 3.39-.46-3.33-.65-6.71-.45-10.07l.87-.75z"/><path fill="#4bacb8" d="M77.15 81.67c1.6 2.2 2.43 4.82 3.28 7.37-1.29.64-2.58 1.26-3.89 1.87-.48-3.08-.36-6.25.61-9.24z"/><path fill="#4bacb8" d="M66.65 83.31c2.03-.5 3.47.02 4.32 1.57-1.59 1.1-3.26 3.04-5.44 2.24.36-1.28.73-2.55 1.12-3.81z"/><path fill="#4bacb8" d="M69.09 93.78c.18-3.03.8-6.12 2.81-8.5 1.01 2.18 1.42 4.54 1.71 6.91-1.49.57-3 1.1-4.52 1.59z"/><path fill="#4bacb8" d="M61.36 86.96c1.54-1.5 3.49 1.06 1.3 1.83-1.57 1.48-3.29-.88-1.3-1.83z"/><path fill="#4bacb8" d="M56.21 91.5c3.65-.62 7.13-1.86 10.49-3.37.36 2.03.63 4.07.84 6.13-8.2 2.23-16.5 4.14-24.92 5.29-.37-.19-1.12-.57-1.49-.77 2.82-1.42 5.03-4.41 8.42-4.34 1.05-.99 2.07-2.01 3.08-3.04 1.19.07 2.38.19 3.58.1z"/><path fill="#4bacb8" d="M105.28 100.24c3.99-2.12 7.57-5.9 12.41-5.61-8.53 14.17-22.49 25.47-38.79 29.17-6.32 1.61-12.88 1.5-19.34 1.35 7.32-2.34 13.71-6.68 20.43-10.28 8.43-4.88 16.78-9.9 25.29-14.63z"/><path fill="#4bacb8" d="M40.35 102.49c10.5-1.22 20.97-3.12 30.99-6.58-1.14 2.71-4.16 3.06-6.54 4.11-10.62 5.13-19.97 12.65-30.98 17.03-2.37-1.42-4.59-3.07-6.82-4.7 4.52-3.19 8.81-6.72 13.35-9.86z"/></svg>
```

Comment:
<img width="83" alt="Screen Shot 2020-09-30 at 9 23 14 AM" src="https://user-images.githubusercontent.com/3766511/94690842-9635f780-02fe-11eb-8cf5-71cadaf925a6.png">

<img width="78" alt="Screen Shot 2020-09-30 at 9 19 14 AM" src="https://user-images.githubusercontent.com/3766511/94690790-874f4500-02fe-11eb-9aed-61917adfe704.png">

I'm not sure how I feel about this transformation. For complex logos, perhaps it's best to leave them as is.

Reply:
Ok I'll leave that as it is!